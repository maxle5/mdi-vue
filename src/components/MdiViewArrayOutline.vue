
  <template>
      <svg :width="size" :height="size" :viewBox="viewbox">
          <path d="M15 7V16H10V7H15M21 5H18V18H21V5M17 5H8V18H17V5M7 5H4V18H7V5Z" style="fill: currentColor" />
      </svg>
  </template>

  <script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiViewArrayOutline",
      props: {
          size: { 
              type: Number, 
              default: 24 
          },
          viewbox: {
              type: String, 
              default: "0 0 24 24" 
          },
      },
  })
  </script>
