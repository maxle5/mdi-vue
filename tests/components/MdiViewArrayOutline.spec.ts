
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiViewArrayOutline from "../../src/components/MdiViewArrayOutline.vue";

test("MdiViewArrayOutline snapshot", () => {
  const wrapper = mount(MdiViewArrayOutline, {});
  expect(wrapper.html()).toMatchSnapshot();
});
