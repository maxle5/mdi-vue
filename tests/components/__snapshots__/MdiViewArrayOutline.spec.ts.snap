// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiViewArrayOutline snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M15 7V16H10V7H15M21 5H18V18H21V5M17 5H8V18H17V5M7 5H4V18H7V5Z" style="fill: currentColor;"></path>
</svg>"
`;
